env.d.ts

```diff
@@ -7,10 +7,34 @@ declare module '*.vue' {
   export default component
 }
 declare global {
-  import type { SmartContractRequestParams } from '@/utils/signerMakeTx'
+  type SmartContractAcceptedValue = (string | undefined | number)[]
+
+  type TransactionSignerParams = {
+    appAddress: string
+    provider: any
```

Comment:
Can we use the `ArcanaProvider` type exported from the auth SDK here?

Reply:
Resolved at https://github.com/arcana-network/developer-dashboard/pull/118/commits/f4c42b36050b13d9928f3ec362b10d5dfaadc9bd

Reply:
`@arcana/auth` doesn't export `ArcanaProvider` type

Reply:
cc: @makylfang

Reply:
Hmm, ill create an interface around it.

Mostly it will be somthing like this
```ts
interface RequestArguments {
  readonly method: string;
  readonly params?: readonly unknown[] | object;
}

interface Provider {
	request(args: RequestArguments): Promise<unknown>
	on(event: string | symbol, listener: fn)
	removeListener(event:string | symbol, listener: fn)
}
```